lib/kenji.rb

```diff
@@ -181,7 +178,13 @@ def respond(code, message, hash={})
     #
     def respond_raw(status = 200, data)
       @status = status
-      body = data.is_a?(IO) ? data : [data]
+
+      body = if data.is_a?(StringIO) || data.kind_of?(IO)
```

Comment:
Hound is right here, use `is_a?`.

IIRC, the Rack spec specifies that the body should be something which response to `each`. So, it shouldn't be a `String`, but probably an `IO` or an `Array` with one string. What's the issue with returning a `StringIO`? Perhaps the test is broken?